packages/front-end/components/Layout/Layout.tsx

```diff
@@ -188,7 +188,10 @@ const navlinks: SidebarLinkProps[] = [
         name: "Webhooks",
         href: "/settings/webhooks",
         path: /^settings\/webhooks/,
-        filter: ({ permissions }) => permissions.check("manageWebhooks"),
+        filter: ({ permissionsUtils }) =>
+          permissionsUtils.canCreateEventWebhook() ||
+          permissionsUtils.canUpdateEventWebhook() ||
+          permissionsUtils.canDeleteEventWebhook(),
```

Comment:
When we add a `canViewEventWebhook()` permission, I think that's the only one we need to check here.